lib/synced/timestamp.rb

```diff
@@ -0,0 +1,9 @@
+class Synced::Timestamp < ActiveRecord::Base
```

Comment:
I think this class is missing `synced_model` attribute, having `scope` only doesn't give as much, when syncing e.g. Rentals and  Bookings at different moments we would need separate records to get the timestamp.